// Copyright 2017 Intel Corporation
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
// ----------------------------------------------------------------------------

syntax = "proto3";

message Device {
	string DeviceIdentity = 1;   
	string TimeFunction = 2;
	string Measurement =3;
	float xmin = 4;
	float xmax = 5;
	float ReliabilityScore = 6;
}

message DevicesList {
	repeated Device Device = 1;
}


